gnucash/import-export/import-main-matcher.c

```diff
@@ -138,11 +145,42 @@ static void refresh_model_row (
                     GNCImportTransInfo *info);
 /* end local prototypes */
 
+static gboolean delete_hash (gpointer key, gpointer value, gpointer user_data)
+{
+    // Value is a hash table that needs to be destroyed. 
+    g_hash_table_destroy (value);
+    return TRUE;
+}
+
+static void
+update_all_balances (GNCImportMainMatcher *info)
+{
+    for (GSList* iter = info->edited_accounts; iter; iter=iter->next)
+    {
+        xaccAccountBeginEdit(iter->data);
+        gnc_account_set_defer_bal_computation(iter->data,0);
+        xaccAccountCommitEdit(iter->data);
```

Comment:
`xaccAccountRecomputeBalance` just returns if the edit level > 0, so calling `xaccAccountBeginEdit` stops it from working.